src/spec/billing_spec.js

```diff
@@ -32,29 +32,15 @@ describe('Billing Page', () => {
               "name": "PLAN-60K" }
           ]
         });
+       $httpBackend.whenGET(/\/resources\/countries\.json/).respond(200, [{"code": "BV","en": "The Bolivian people","es": "Bolivia"}]); 
```

Comment:
No, https://github.com/MakingSense/Doppler_Relay/blob/develop/Relay.Application.Api/wwwroot/resources/countries.json#L172 I change the name of country, thanks